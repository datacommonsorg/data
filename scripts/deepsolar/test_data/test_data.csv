block_group_FIPS,num_of_installations_residential,avg_size_residential,num_of_installations_solar_heat,avg_size_solar_heat,num_of_installations_commercial,avg_size_commercial,num_of_installations_utility_scale,avg_size_utility_scale,num_of_installations
490351135232,3,29.571202016666664,0,0.0,1,160.062938847,0,0.0,4
271310705012,3,40.080141905733335,1,88.75645322870001,0,0.0,0,0.0,4
170318008001,2,22.9637802954,0,0.0,1,211.873296645,0,0.0,3
360470696022,11,18.323446394659094,0,0.0,0,0.0,0,0.0,11
490111271001,1,10.4879130204,0,0.0,0,0.0,0,0.0,1
40138144002,16,39.70766259088125,1,28.6313617389,0,0.0,0,0.0,17
190610008011,1,47.3413584739,0,0.0,0,0.0,0,0.0,1
190539603001,0,0.0,0,0.0,2,258.3234168673,0,0.0,2
260992602002,0,0.0,1,21.7220316923,0,0.0,0,0.0,1
60590626394,4,15.892544460675,0,0.0,1,1037.81172854,0,0.0,5
271630707041,1,100.134279945,0,0.0,0,0.0,0,0.0,1
121090207111,3,18.083656593616663,4,20.59256548755,0,0.0,0,0.0,7
250214192002,1,21.0544043953,0,0.0,0,0.0,0,0.0,1
410390050001,6,17.888506099335,0,0.0,0,0.0,0,0.0,6
60730177022,27,28.65239110572296,0,0.0,1,282.172455705,0,0.0,28
250092219011,7,27.15583592318572,0,0.0,0,0.0,0,0.0,7
360690501012,3,22.8348433014,0,0.0,10,473.11954818907,0,0.0,13
120310149013,0,0.0,3,13.39717012191,0,0.0,0,0.0,3
350150006004,0,0.0,0,0.0,1,422.90796795,0,0.0,1
401430073081,1,102.684208682,0,0.0,0,0.0,0,0.0,1
60855091021,5,112.051580275794,1,10.7027971012,14,320.0682744183114,0,0.0,20
130730303042,0,0.0,1,25.4117079812,0,0.0,0,0.0,1
530330014003,4,32.338535551775,0,0.0,0,0.0,0,0.0,4
240178507131,105,37.28852248244457,0,0.0,0,0.0,0,0.0,105
450790113052,5,34.39551247786,0,0.0,0,0.0,0,0.0,5
360750211032,2,26.9472792339,0,0.0,2,193.8718091915,0,0.0,4
240338010061,20,36.5446629762985,0,0.0,0,0.0,0,0.0,20
290210028005,0,0.0,0,0.0,1,179.229565715,0,0.0,1
60374333021,1,33.0672794044,0,0.0,0,0.0,0,0.0,1
40133199051,3,62.38766333123334,0,0.0,0,0.0,0,0.0,3
999999999999,None,None,None,None,None,None,None,None,None
