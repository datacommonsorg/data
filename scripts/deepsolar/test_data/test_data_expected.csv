block_group_FIPS,num_of_installations_residential,avg_size_residential,num_of_installations_solar_heat,avg_size_solar_heat,num_of_installations_commercial,avg_size_commercial,num_of_installations_utility_scale,avg_size_utility_scale,num_of_installations
dcid:geoId/490351135232,3.0,29.571202016666664,0.0,0.0,1.0,160.062938847,0.0,0.0,4.0
dcid:geoId/271310705012,3.0,40.080141905733335,1.0,88.75645322870001,0.0,0.0,0.0,0.0,4.0
dcid:geoId/170318008001,2.0,22.9637802954,0.0,0.0,1.0,211.873296645,0.0,0.0,3.0
dcid:geoId/360470696022,11.0,18.323446394659094,0.0,0.0,0.0,0.0,0.0,0.0,11.0
dcid:geoId/490111271001,1.0,10.4879130204,0.0,0.0,0.0,0.0,0.0,0.0,1.0
dcid:geoId/040138144002,16.0,39.70766259088125,1.0,28.6313617389,0.0,0.0,0.0,0.0,17.0
dcid:geoId/190610008011,1.0,47.3413584739,0.0,0.0,0.0,0.0,0.0,0.0,1.0
dcid:geoId/190539603001,0.0,0.0,0.0,0.0,2.0,258.3234168673,0.0,0.0,2.0
dcid:geoId/260992602002,0.0,0.0,1.0,21.7220316923,0.0,0.0,0.0,0.0,1.0
dcid:geoId/060590626394,4.0,15.892544460675,0.0,0.0,1.0,1037.81172854,0.0,0.0,5.0
dcid:geoId/271630707041,1.0,100.134279945,0.0,0.0,0.0,0.0,0.0,0.0,1.0
dcid:geoId/121090207111,3.0,18.083656593616663,4.0,20.59256548755,0.0,0.0,0.0,0.0,7.0
dcid:geoId/250214192002,1.0,21.0544043953,0.0,0.0,0.0,0.0,0.0,0.0,1.0
dcid:geoId/410390050001,6.0,17.888506099335,0.0,0.0,0.0,0.0,0.0,0.0,6.0
dcid:geoId/060730177022,27.0,28.65239110572296,0.0,0.0,1.0,282.172455705,0.0,0.0,28.0
dcid:geoId/250092219011,7.0,27.15583592318572,0.0,0.0,0.0,0.0,0.0,0.0,7.0
dcid:geoId/360690501012,3.0,22.8348433014,0.0,0.0,10.0,473.11954818907,0.0,0.0,13.0
dcid:geoId/120310149013,0.0,0.0,3.0,13.39717012191,0.0,0.0,0.0,0.0,3.0
dcid:geoId/350150006004,0.0,0.0,0.0,0.0,1.0,422.90796795,0.0,0.0,1.0
dcid:geoId/401430073081,1.0,102.684208682,0.0,0.0,0.0,0.0,0.0,0.0,1.0
dcid:geoId/060855091021,5.0,112.051580275794,1.0,10.7027971012,14.0,320.0682744183114,0.0,0.0,20.0
dcid:geoId/130730303042,0.0,0.0,1.0,25.4117079812,0.0,0.0,0.0,0.0,1.0
dcid:geoId/530330014003,4.0,32.338535551775,0.0,0.0,0.0,0.0,0.0,0.0,4.0
dcid:geoId/240178507131,105.0,37.28852248244457,0.0,0.0,0.0,0.0,0.0,0.0,105.0
dcid:geoId/450790113052,5.0,34.39551247786,0.0,0.0,0.0,0.0,0.0,0.0,5.0
dcid:geoId/360750211032,2.0,26.9472792339,0.0,0.0,2.0,193.8718091915,0.0,0.0,4.0
dcid:geoId/240338010061,20.0,36.5446629762985,0.0,0.0,0.0,0.0,0.0,0.0,20.0
dcid:geoId/290210028005,0.0,0.0,0.0,0.0,1.0,179.229565715,0.0,0.0,1.0
dcid:geoId/060374333021,1.0,33.0672794044,0.0,0.0,0.0,0.0,0.0,0.0,1.0
dcid:geoId/040133199051,3.0,62.38766333123334,0.0,0.0,0.0,0.0,0.0,0.0,3.0
dcid:geoId/999999999999,,,,,,,,,
